static/js/failed.js

```diff
@@ -0,0 +1,202 @@
+$(function () {
```

Comment:
Is there a difference between what this script does and https://github.com/andela/healthchecks_akira/blob/df137bfdad57cb53bcf31bdc92056cb4f316f921/static/js/checks.js?

We should consider re-using checks.js if not.